webpack.config.js

```diff
@@ -42,3 +34,25 @@ module.exports = {
         })
     ]
 };
+
+module.exports = [Object.assign({}, base, {
+    entry: {
+        'render': './src/index-web.js',
+        'render.min': './src/index-web.js'
+    },
+    output: {
+        path: __dirname,
+        filename: '[name].js'
+    },
+}),
+Object.assign({}, base, {
+    entry: {
+        'render': './src/index.js'
+    },
+    output: {
+        library: 'ScratchRender',
+        libraryTarget: 'commonjs2',
```

Comment:
Apparently you can give webpack an array of configurations and it will build multiple targets. Since the output of dist.js isn't a web bundle, I couldn't just add it as an entry to the existing config and had to do this.